.editorconfig

```diff
@@ -0,0 +1,8 @@
+[*]
+end_of_line = lf
+insert_final_newline = true
+indent_style = space
+indent_size = 4
+charset = utf-8
+trim_trailing_whitespace = true
+max_line_length = 80
```

Comment:
I don't think this file really goes in source control - it's specific to your editor I believe.